client/components/file-upload/__tests__/index.test.tsx

```diff
@@ -108,10 +108,11 @@ describe( 'FileUploadControl', () => {
 			'input[type="file"]'
 		);
 		if ( input !== null ) {
-			await userEvent.upload( input, file );
-			await userEvent.upload( input, file );
```

Comment:
Not sure why exactly, but switching to `fireEvent` seemed to have fixed the issue 🤷